### ISSUE TITLE
[bug] Some localized text not showing anymore + slowness

### ISSUE THREAD

**Issue by WarningImHack3r** [inlang/inlang#1228]
### Did you check if the config file includes errors?

- [X] Yes, I have validated the config file

### Problem

Since my last issue (#1195), I updated without issue (for now, let's hope #1197 doesn't happen again) to the latest **0.11.4**.
I made the mistake of only relying on CI and build status to validate and push the website to prod, without actually opening the dev server in the Dependabot branch.

However, I had some stuff to add today and I fired the dev server on my up-to-date-with-main branch, and I noticed 2 things:
- The website's main title (`<h1>`) is not showing anymore in French, but does actually work if I switch to English, but still not if I switch back to French. It's replaced by an `<br>`, that's it.
- Second, on page load/refresh, I noticed that the website's text takes a few ms to appear, which never was the case before when I was sticking to `0.6.3`/`0.7.1`.

So I'm going to push a quick fix to downgrade again to `0.7.1` because this version doesn't have those bugs and builds correctly (despite randomly not building in the CI). I will only ever upgrade again when both the build pass and the website doesn't have any UI regression from this version.

### Expected behavior

Not to have these bugs, work as well as it did in the old version I used to be in.

### Reproduction

Build and start the dev server, and optionally play with the footer language toggle switch.
Source code is available [here](https://github.com/EmeraldHQ/Website) ([permalink before downgrade](https://github.com/EmeraldHQ/Website/tree/1bfd559d0470f11e74519a1085623765c693c0e1)).

### System Info

```Shell
System:
    OS: macOS 13.5
    CPU: (10) arm64 Apple M1 Pro
    Memory: 65.09 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.5.0 - /opt/homebrew/bin/node
    npm: 9.8.1 - /opt/homebrew/bin/npm
    pnpm: 8.6.12 - /opt/homebrew/bin/pnpm
  Browsers:
    Chrome: 114.0.5735.198
    Safari: 16.6
    Safari Technology Preview: 17.0
  npmPackages:
    @inlang/sdk-js: ^0.11.4 => 0.11.4 
    @sveltejs/adapter-static: ^2.0.3 => 2.0.3 
    @sveltejs/kit: ^1.22.4 => 1.22.4 
    svelte: ^4.1.2 => 4.1.2
```

**Comment by ivanhofer** [comment]
Thanks for the link to the repository!

The issues should be fixed in version `0.11.6`.

The problem to the missing strings was that `Svelte` has executed the `i` function before it was initialized. This happens when using `@html i()` in your markup. Once the `i` function was initialized correctly, the `@html` statement was not re-executed. We have updated the code to not render anything util everything is initialized.
This is also the fix to the second problem. Inlang now does not render empty strings but instead delays the rendering of the whole page.

The combination of a static page and loading the language on the client is something we didn't account before. Now it should work as expected.

**Comment by WarningImHack3r** [comment]
Thanks for the good work Ivan!!

**Comment by WarningImHack3r** [comment]
@ivanhofer cannot build: https://github.com/EmeraldHQ/Website/actions/runs/5832754170/job/15818829441?pr=78

**Comment by ivanhofer** [comment]
Sorry, I swear I have tested it locally and it did work ^^. I will take another look..

**Comment by WarningImHack3r** [comment]
Out of curiosity: why don't you only work with PRs to systematically run a `build` workflow (you seem to already have), so you don't release a broken version accidentally?

**Comment by ivanhofer** [comment]
Dammit, the only part that had no types, did't show an error after refactoring ^^.
I have released version `0.11.7` now.

> Out of curiosity: why don't you only work with PRs to systematically run a `build` workflow (you seem to already have), so you don't release a broken version accidentally?

The issue is happening during runtime and not build time. We plan to have e2e tests for the future to catch those bugs. But right now we don't have the capacity to set them up.

**Comment by WarningImHack3r** [comment]
Okay, that makes sense! Thanks for your reactivity :)
